how-to/workspace-platform-starter/client/src/framework/auth.ts

```diff
@@ -1,7 +1,7 @@
 import { createLogger } from "./logger-provider";
 import { initializeModule, loadModules } from "./modules";
-import type { ModuleHelpers } from "./shapes";
 import type { AuthProvider, AuthProviderOptions } from "./shapes/auth-shapes";
+import type { ModuleHelpers } from "./shapes/module-shapes";
```

Comment:
why does auth need module helpers?

Reply:
Thats from before, I just update the import - it needs to initialize the loaded auth modules with the helpers